core-processor/src/ext.rs

```diff
@@ -180,6 +175,7 @@ pub struct Ext {
 /// Empty implementation for non-substrate (and non-lazy-pages) using
 impl ProcessorExt for Ext {
     type Error = ExtError;
+    const IS_LAZY_PAGES_ENABLED: bool = false;
```

Comment:
```suggestion
    const LAZY_PAGES_ENABLED: bool = false;
```

for me question-like constants seems strange

Reply:
Done